packages/create-vite/src/index.ts

```diff
@@ -225,7 +225,7 @@ async function init() {
           },
         },
         {
-          type: () =>
+          type:
             !fs.existsSync(targetDir) || isEmpty(targetDir) ? null : 'confirm',
```

Comment:
`targetDir` value will change after the first prompt. So removing `() =>` will break the functionality. Other places are same as well.

Reply:
Oops, probably should add a test for it then? seems like all tests passed?